SleepEngine/Engine/GameWindow.cpp

```diff
@@ -84,38 +69,10 @@ bool GameWindow::shouldClose() const
 
 void GameWindow::runFrame()
 {
-    EASY_FUNCTION(profiler::colors::Orange);
-    m_clock.frameStart();
-
-    update();
-    render();
-
-    m_renderer->render();
-
     EASY_BLOCK("Swap buffers/poll events", profiler::colors::Purple);
```

Comment:
Since you deleted the EASY_FUNCTION code above, you should change this line to EASY_FUNCTION.
**And delete EASY_END_BLOCK below!**
```suggestion
    EASY_FUNCTION(profiler::colors::Orange);
```